modules/dreamview/backend/simulation_world/simulation_world_service.cc

```diff
@@ -550,11 +553,25 @@ Object &SimulationWorldService::CreateWorldObjectIfAbsent(
     Object &world_obj = obj_map_[id];
     SetObstacleInfo(obstacle, &world_obj);
     SetObstaclePolygon(obstacle, &world_obj);
-    SetObstacleType(obstacle, &world_obj);
+    SetObstacleType(obstacle.type(), obstacle.sub_type(), &world_obj);
+    SetObstacleSensorMeasurements(obstacle, &world_obj);
   }
   return obj_map_[id];
 }
 
+void SimulationWorldService::CreateWorldObjectFromSensorMeasurement(
+    const SensorMeasurement &sensor, Object *sensormeasure_object) {
+  // set sensor measurement info, sensor measurement theta——>object heading
```

Comment:
the code and the function name is clear enough, might not need the comment here.